src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -77,6 +113,7 @@ private void AddOrRemoveDropdown(bool enabled)
                 var document = _languageService.EditorAdaptersFactoryService.GetDataBuffer(buffer).AsTextContainer().GetRelatedDocuments().FirstOrDefault();
                 if (document.GetLanguageService<INavigationBarItemService>() == null)
```

Comment:
What happens if document is null here? Is this missing a ? in document?.GetLanguageService()...?

(it looks like it'll nicely remove the drop down in that case, so that's good)

Reply:
GetLanguageService puts a ? on document - http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/DocumentExtensions.cs,33

I can make this more explicit though and check that the document is null.